packages/url_launcher/url_launcher_macos/android/build.gradle

```diff
@@ -0,0 +1,34 @@
+group 'io.flutter.plugins.url_launcher_macos'
+version '1.0'
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.0'
+    }
+}
+
+rootProject.allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        minSdkVersion 16
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+    lintOptions {
+        disable 'InvalidPackage'
+    }
```

Comment:
In otherr plugins I had to add an androidx lint dependency here. Did you not have this error locally?

Reply:
Which error did you run into?
The url_launcher example apk build locally for me...

Reply:
It is possible that I was getting them on 1.9 so it might not be an issue anymore. :)